main.js

```diff
@@ -87,11 +87,15 @@ var app = {
   },
   details: {
     item: null
+  },
+  cart: {
+    items: []
   }
 }
 
 var renderCatalog = document.querySelector("[data-view='catalog']")
 var renderDetails = document.querySelector("[data-view='details']")
+var renderCart = document.querySelector("[data-view='cart']")
```

Comment:
We'll be dealing with the cart view in a couple issues. You can remove this variable for now.